src/backend/base/langflow/components/processing/dataframe_operations.py

```
@@ -137,36 +164,61 @@
             build_config[field]["show"] = False
 
         if field_name == "operation":
-            if field_value == "Filter":
+            # Handle SortableListInput format
+            if isinstance(field_value, list):
+                if len(field_value) > 0:
+                    operation_name = field_value[0].get("name", "")
+                else:
+                    # Handle empty selection (user deselected)
+                    operation_name = ""
```

Ruff (0.11.9)

Use ternary operator `operation_name = field_value[0].get("name", "") if len(field_value) > 0 else ""` instead of `if`-`else`-block

(SIM108)